api/migrations/README

```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```

Comment:
Please also delete `/api/migrations/` and `/api/popUpInfo/` while you're at it. `app.db` and `config.py` should also be deleted.